cleanlab/multilabel_classification/dataset.py

```diff
@@ -0,0 +1,314 @@
+    for i in issues_dict:
+        issues_df_dict["Class Index"].append(i)
+        for j in issues_dict[i]:
+            issues_df_dict[j].append(issues_dict[i][j])
+    return pd.DataFrame.from_dict(issues_df_dict).sort_values(
+        by="Label Quality Score", ascending=True
+    )
+
+
+def _get_num_examples_multilabel(labels=None, confident_joint: Optional[np.ndarray] = None) -> int:
+    """Helper method that finds the number of examples from the parameters or throws an error
+    if neither parameter is provided.
+
+    Parameters
+    ----------
+    For parameter info, see the docstring of :py:func:`common_multilabel_issues <cleanlab.multilabel_classification.dataset.common_multilabel_issues>`
+
+    Returns
+    -------
+    num_examples : int
+        The number of examples in the dataset.
+
+    Raises
+    ------
+    ValueError
+        If `labels` is None."""
+
+    if labels is None and confident_joint is None:
+        raise ValueError(
+            "Error: num_examples is None. You must either provide confident_joint, "
+            "or provide both num_example and joint as input parameters."
+        )
+    _confident_joint = cast(np.ndarray, confident_joint)
+    num_examples = len(labels) if labels is not None else cast(int, np.sum(_confident_joint[0]))
+    return num_examples
+
+
+def overall_multilabel_health_score(
+    labels=None,
+    pred_probs=None,
+    *,
+    joint=None,
+    confident_joint=None,
+) -> float:
+    """Returns a single score between 0 and 1 measuring the overall quality of all labels in a multi-label classification dataset.
+    Intuitively, the score is the average correctness of the given labels across all examples in the
+    dataset. So a score of 1 suggests your data is perfectly labeled and a score of 0.5 suggests
+    half of the examples in the dataset may be incorrectly labeled. Thus, a higher
+    score implies a higher quality dataset.
+
+    This method works by providing ``labels``, ``pred_probs`` and an optional Confident Joint.
+
+
+    Only provide **exactly one of the above input options**, do not provide a combination.
+
+    **Parameters**: For parameter info, see the docstring of :py:func:`find_overlapping_classes <cleanlab.dataset.find_overlapping_classes>`.
+
+    Returns
+    -------
+    health_score : float
+        A score between 0 and 1, where 1 implies all labels in the dataset are estimated to be correct.
+        A score of 0.5 implies that half of the dataset's labels are estimated to have issues.
+    """
+    num_classes = get_num_classes(labels=labels, pred_probs=pred_probs, multi_label=True)
+    class_names = list(range(num_classes))
+    num_examples = _get_num_examples_multilabel(labels=labels, confident_joint=confident_joint)
+    issues_df = common_multilabel_issues(
+        labels=labels, pred_probs=pred_probs, class_names=class_names, confident_joint=joint
+    )
+    return sum(issues_df["Num Examples"]) / num_examples
```

Comment:
this math does not seem correct? arent the rows of DF returned by `common_multilabel_issues` not disjoint at all? So you could get a score here that is greater than 1?

I think this method should call `issues = multilabel_classification.find_label_issues()` and then report the number of issues divided by num examples.

Please ensure you add a unit test for this method where you introduce only 2 label issues in a multi-label dataset with 100 examples and you verify this method returns 0.02